.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: Backpack CI
+
+on:
+  push:
```

Comment:
What about tags? Should we run on those too?(in which case the checks with `refs/head/master` will need to be tweaked as the ref will be `refs/tags/x.y.z`

Reply:
Sure thing! Wasn't sure about this one as when we publish tags we publish multiple ones (one for each package) so not sure how that will play with the deploy based on tags as we don't want to kick off 30 builds at once 🤔

Reply:
That is a good point 🤔 Probably wouldn't be great if that happened

Reply:
Yeah we need to think how to do this responsibly - I took the root of just doing pushes based on master as we don't tend to make changes directly on master without them going through a PR so means if we are merging they should be good to deploy the docs straight away?